third_party/vmServiceDrivers/org/dartlang/vm/service/VmService.java

```diff
@@ -151,11 +153,11 @@ public void clearCpuProfile(String isolateId, SuccessConsumer consumer) {
   }
 
   /**
-   * @undocumented
+   * Clears all VM timeline events.
    */
   public void clearVMTimeline(SuccessConsumer consumer) {
     final JsonObject params = new JsonObject();
-    request("_clearVMTimeline", params, consumer);
+    request("clearVMTimeline", params, consumer);
```

Comment:
Deferring to @kenzieschmoll on these protocol changes...

Reply:
These calls aren't used by the Flutter IntelliJ client, so should be no-ops in terms of the change.